frontend/src/shared/components/left-sidebar/left-sidebar.component.sass

```diff
@@ -0,0 +1,70 @@
+@import 'src/colors.sass'
+.left-sidebar
+  width: 252px
+  height: 100%
+  position: fixed
+  z-index: 1
+  overflow-x: hidden
+  left: 0
+  top: 0
+  background: colorscheme(white-100)
+  box-shadow: 3px 4px 9px rgba(223, 223, 223, 0.25)
+
+.logo
+  background: colorscheme(gray-200)
+  width: 87.63px
+  height: 57.23px
+  border-radius: 10px
+  margin-left: 51px
+  margin-top: 32.34px
+
+  display: flex
+  align-items: center
+  justify-content: center
+
+.logo-text
+  vertical-align: middle
+
+.icon
+  margin-left: 50px
+  margin-top: 76.45px
+
+.menu
+  margin-left: 49px
+  margin-top: 48.01px
+  margin-right: 48px
```

Comment:
and here there is none?

Reply:
Added whitespace

Reply:
It's good to have this setting enabled in vs code to not bother anymore with white spaces
![image](https://user-images.githubusercontent.com/103319266/183685661-f2138342-3f78-4485-a5a5-dc4c4ff8943c.png)